public/app/features/alerting/unified/utils/misc.ts

```diff
@@ -35,6 +35,16 @@ export function createExploreLink(dataSourceName: string, query: string) {
   });
 }
 
+export function createContactPointLink(contactPoint: string, alertManagerSourceName = ''): string {
+  return `/alerting/notifications/receivers/${encodeURIComponent(contactPoint)}/edit?alertmanager=${encodeURIComponent(
```

Comment:
Please use `createUrl` helper that takes care of the case when Grafana is hosted on a subpath